### DISCUSSION TITLE
Use different last.fm APIKEY between different users

### DISCUSSION THREAD

**Discussion by haha4ni** [navidrome/navidrome#3465] [General]
I searched the discussions and couldn't find any similar questions.
Navidrome seems to share the same set of Last.fm APIKEY. I hope different users can use their own APIKEY. Or does this feature already exist?

**Comment by deluan** [comment]
Sorry for the delay. 

The APIKey is for the server, not for individual users. After the APIKey and Secret are properly configured, each user has to authorize the server to send scrobbles to their user, by going to the "Personal" view in Navidrome's UI and clicking on "Scrobble to Last.fm"

![Screenshot 2024-12-18 at 1 58 07 PM](https://github.com/user-attachments/assets/adf02b2b-9929-4e71-ae42-6ac88ffac73d)

That way each user will have their own scrobbles.

**Comment by haha4ni** [comment]
Thank you! I have completely figured it out, and using it with friends has been fantastic! It has been really nice to use so far. The recent update has made album management even more flexible. Awesome!
